xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -910,7 +910,12 @@ JSONRPC_STATUS CVideoLibrary::Export(const std::string &method, ITransportLayer
 
 JSONRPC_STATUS CVideoLibrary::Clean(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
 {
-  std::string cmd = StringUtils::Format("cleanlibrary(video, %s)", parameterObject["showdialogs"].asBoolean() ? "true" : "false");
+  std::string cmd;
+  if (parameterObject["content"].empty())
+    cmd = StringUtils::Format("cleanlibrary(video, %s)", parameterObject["showdialogs"].asBoolean() ? "true" : "false");
+  else
+    cmd = StringUtils::Format("cleanlibrary(%s, %s)", parameterObject["content"].asString().c_str(), parameterObject["showdialogs"].asBoolean() ? "true" : "false");
```

Comment:
same here and no need to call `c_str()` `StringUtils::Format` uses `fmt` so it handles complex types fine

Reply:
done.